recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - pip
   run:
     - python
-    - vaex-core >=1.3.0,<2
+    - vaex-core >=2.0.0,<2
```

Comment:
I don't know if that will result in a solvable env, there aren't any versions that can satisfy that constraint.

Reply:
yeah, I need sleep :)